sdk/sdktypes/project.go

```diff
@@ -19,7 +19,7 @@ var (
 )
 
 func strictValidateProject(pb *projectv1.Project) error {
-	if err := ensureNotEmpty(pb.Name, pb.ProjectId); err != nil {
+	if err := ensureNotEmpty(pb.Name, pb.ProjectId, pb.ResourcesRootUrl); err != nil {
```

Comment:
why do we ensure this is not empty if we support not having ResourcesRootUrl ?

Reply:
no longer relevant